examples/hello-world/step-by-step/cifar10/stats/image_stats.ipynb

```diff
@@ -552,6 +480,7 @@
    },
    "outputs": [],
    "source": [
+    "! python image_stats_job.py -co -j /tmp/nvflare/jobs/stats_image -n 2\n",
     "! nvflare simulator /tmp/nvflare/jobs/stats_image -w /tmp/nvflare/image_stats -n 2 -t 2\n"
```

Comment:
instead of saying "Run Job using Simulator CLI"
maybe we just focus on "Export job to be run in simulator, POC or Production"?